### DISCUSSION TITLE
MAC Installation

### DISCUSSION THREAD

**Discussion by markgabay** [dotnet/maui#4577] [Q&A] [answered]
I am trying to follow the installation and I am getting this error

Workload installation failed: microsoft.netcore.app.runtime.mono.android-arm::6.0.2-mauipre.1.22102.15 is not found in NuGet feeds https://api.nuget.org/v3/index.json".

**Comment by linus-ekstrom** [comment, marked as answer]
Had the same issue. Try running the command with these extra parameters:
```
dotnet workload install maui `
   --from-rollback-file https://aka.ms/dotnet/maui/preview.12.json `
   --source https://aka.ms/dotnet6/nuget/index.json `
   --source https://api.nuget.org/v3/index.json
```

This successfully installed the maui workload for me.

**Reply by markgabay** [reply]
Hey @linus-ekstrom , it worked!! Thank you!

**Reply by jsuarezruiz** [reply]
Just in case someone is in the same situation, this Wiki explaining how to install in macOS could help: https://github.com/dotnet/maui/wiki/macOS-Install

**Reply by MrCamara** [reply]
Worked for me too. But I hope it is fully integrate in VS for Mac in Q2 so we don't have to do it by command line anymore....
